src/pages/Facility/settings/organizations/components/FacilityOrganizationLayout.tsx

```diff
@@ -82,31 +93,72 @@ export default function FacilityOrganizationLayout({
   }
 
   return (
-    <Page title={`${org.name} `}>
-      {/* Navigation */}
-      <div className="mt-4">
-        <Menubar>
-          {navItems.map((item) => (
-            <MenubarMenu key={item.path}>
-              <MenubarTrigger
-                className={`${
-                  path === item.path
-                    ? "font-medium text-primary-700 bg-gray-100"
-                    : "hover:text-primary-500 hover:bg-gray-100 text-gray-700"
-                }`}
-                asChild
-              >
-                <Link href={item.path} className="cursor-pointer">
-                  <CareIcon icon={item.icon} className="mr-2 h-4 w-4" />
-                  {item.title}
-                </Link>
-              </MenubarTrigger>
-            </MenubarMenu>
-          ))}
-        </Menubar>
+    <>
+      <div className="md:px-6 py-2 flex items-center gap-2 mx-auto max-w-4xl">
+        <Breadcrumb>
```

Comment:
<img width="372" alt="image" src="https://github.com/user-attachments/assets/0e3833f2-d4c3-4fd3-aed6-0736d3440128" />

Is breadcrumbs necessary if it's same level as root? 🤔